esp-wifi/src/ble/os_adapter_esp32.rs

```diff
@@ -397,13 +397,13 @@ pub(crate) unsafe extern "C" fn coex_bt_wakeup_request_end() {
 #[allow(unused_variables)]
 pub(crate) unsafe extern "C" fn coex_bt_request(event: u32, latency: u32, duration: u32) -> i32 {
     trace!("coex_bt_request");
-    extern "C" {
+    unsafe extern "C" {
         #[cfg(coex)]
         fn coex_bt_request(event: u32, latency: u32, duration: u32) -> i32;
     }
 
     #[cfg(coex)]
-    return coex_bt_request(event, latency, duration);
+    return unsafe { coex_bt_request(event, latency, duration) };
```

Comment:
Completely separate from the main PR, but are these going to end up calling themselves recursively? I'm guessing not, given that coex is working, but this looks a bit weird :sweat_smile:. Any ideas @bjoernQ?

Reply:
Looks weird and somewhat confusing, true. It will call the function from the blobs here - the real exported name of this Rust function is different from what the "extern" block imports

Reply:
Thanks, that makes sense.